lib/chef-dk/policyfile_lock.rb

```diff
@@ -15,7 +16,7 @@
 # limitations under the License.
 #
 
-require 'digest/sha1'
+require 'digest/sha2'
```

Comment:
Yes, though that would change anyway with the introduction of attributes, since now there are fields for those that didn't exist in the past. Given the numerous feature flags required to enable the feature and the "beta quality" warnings emitted all over the place, now is a good time to do this, otherwise I'd definitely work on some sort of compatibility process.